Twirf 1: First batch, local communities and CfP process explained

dns2utf8 opened the pull request:
- [x] @pnkfelix please check your team entry and picture
- [x] @GuillaumeGomez and @pnkfelix french rust groups
- [x] First ticket batch
- [x] CfP process explained
- [x] Mention "Paris Salles des Fetes for rust impl days"

dns2utf8 committed:
Fix typo

dns2utf8 committed:
Added @pnkfelix

dns2utf8 committed:
First draft

GuillaumeGomez commented on _posts/2018-03-02-this-week-in-rustfest-1-anonymouse-abstract-first-batch-locals.md:
```diff
@@ -0,0 +1,22 @@
+  - pnkfelix
+---
+
+With some further adoo, this weeks news.
```
"adoo"?

dns2utf8 replied:
like the opposite of "without further ado"

GuillaumeGomez commented on _posts/2018-03-02-this-week-in-rustfest-1-anonymouse-abstract-first-batch-locals.md:
```diff
@@ -0,0 +1,22 @@
+
+# Local communities
+
+@GuillaumeGomez and @pnkfelix
```
Yeay!

dns2utf8 committed:
some more words

GuillaumeGomez committed:
Fix some typos

pnkfelix commented:
Entry looks fine to me. If you want something to fill in that `desc` field, I'll be happy to put up a PR with that after this lands.
 * Update: Oh, *no one* has anything but `""` for their `desc` field. 😆

GuillaumeGomez commented:
@pnkfelix: For now, I thin you should just push to the branch (like I did).

pnkfelix committed:
While fixing a small typo ("famouse" => "famous"), decided to flesh out the answer.

pnkfelix committed:
Revise wording slightly to make intent clearer that anonymity is not a boolean flag.

pnkfelix committed:
Switch to using an item list for the two parts of the response.
(Is this good style? Feel free to revert if its not.)

pnkfelix committed:
Atttempt to start fleshing out the "local community" section. But maybe this is going in wrong direction.

pnkfelix committed:
fix grammar *and* simplify text all at once.

pnkfelix committed:
Ahh! I almost wrote the complete opposite of what I intended!!

pnkfelix committed:
mention snips and scality as industrial users located in paris.

GuillaumeGomez commented on _posts/2018-03-02-this-week-in-rustfest-1-anonymouse-abstract-first-batch-locals.md:
```diff
@@ -0,0 +1,56 @@
+  - pnkfelix
+---
+
+With some further adoo, this weeks news.
```
"this weeks" -> shouldn't it be "this week" instead?

GuillaumeGomez committed:
Flesh out local communities a bit

dns2utf8 committed:
Explain the process and define stuff

pnkfelix committed:
Some grammar fixes.

- The original "This weeks news" was close to correct; it just needed
  an apostrophe.

- Some adjectives needed to be reordered to sound right to my ear.

dns2utf8 committed:
Mention impl days

Hoverbear commented on _posts/2018-03-02-this-week-in-rustfest-1-anonymouse-abstract-first-batch-locals.md:
```diff
@@ -0,0 +1,83 @@
+
+# Local communities
+
+TODO: @GuillaumeGomez and @pnkfelix
```
This can go =D

Hoverbear commented on _posts/2018-03-02-this-week-in-rustfest-1-anonymouse-abstract-first-batch-locals.md:
```diff
@@ -0,0 +1,83 @@
+
+## The information
+
+Once the proposals are voted on, we will start to inform the submitteesi within a week.
```
`submitteesi` is not a word. :(

dns2utf8 replied:
true, using "proposers"

dns2utf8 committed:
Cleanup

dns2utf8 commented:
LGTM, feel free to merge

GuillaumeGomez commented:
Yeay!
